filament/backend/src/opengl/OpenGLDriver.cpp

```diff
@@ -1808,9 +1808,9 @@ void OpenGLDriver::updateCubeImage(Handle<HwTexture> th, uint32_t level,
 
     GLTexture* t = handle_cast<GLTexture *>(th);
     if (data.type == PixelDataType::COMPRESSED) {
-        setCompressedTextureData(t, level, 0, 0, 0, 0, 0, 0, std::move(data), &faceOffsets);
+        setCompressedTextureData(t, level, 0, 0, 0, t->width >> level, t->height >> level, 0, std::move(data), &faceOffsets);
```

Comment:
This doesn't work because `x >>  level` could be 0 when it needs to be 1.  We should use `max(1, x>>level)` instead, like it is done in other places.

Reply:
Done in [a80174417c67a87ecb82d7a68197e5706cebf81f](https://github.com/google/filament/pull/5018/commits/a80174417c67a87ecb82d7a68197e5706cebf81f)